src/main/java/org/kainos/ea/db/AuthDao.java

```diff
@@ -0,0 +1,57 @@
+package org.kainos.ea.db;
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.kainos.ea.cli.Login;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.Date;
+import java.util.UUID;
+
+public class AuthDao
+{
+    private DatabaseConnector databaseConnector = new DatabaseConnector();
+
+
+    public boolean validLogin(Login login) {
+        try (Connection c = databaseConnector.getConnection()) {
+            String getLogin = "SELECT Password FROM `User` WHERE Username = ? ";
+            PreparedStatement st = c.prepareStatement(getLogin, Statement.RETURN_GENERATED_KEYS);
+
+            st.setString(1, login.getUsername());
+
+            ResultSet rs = st.executeQuery();
+
+            while (rs.next()) {
+                return rs.getString("password").equals(login.getPassword());
```

Comment:
Seems like our passwords are stored in plain text, we may want to look into hashing in case the database got leaked.